packages/wonder-blocks-link/src/components/__docs__/link.stories.js

```diff
@@ -307,4 +477,8 @@ const styles = StyleSheet.create({
         flexDirection: "row",
         alignItems: "center",
     },
+    standaloneLinkWrapper: {
+        display: "inline-block",
```

Comment:
I used `inline-block` so that the outline wraps just the text instead of taking up the full width. I tried using `flex` and it doesn't fix the full width outline issue. I'll add a comment to clarify.

<img width="633" alt="Screen Shot 2023-02-09 at 2 58 32 PM" src="https://user-images.githubusercontent.com/13231763/217958272-3a4c6e51-dc37-42c3-8d90-40a0cdb7e6a8.png">
<img width="1536" alt="Screen Shot 2023-02-09 at 2 57 45 PM" src="https://user-images.githubusercontent.com/13231763/217958279-cb88e2c3-d360-4db9-9534-3db66d859369.png">